tests/fixtures/internal/checker/scoped_library.txt

```diff
@@ -0,0 +1 @@
+3:11 TOR901 Use `torch.library._scoped_library` instead of `torch.library.Library` in PyTorch tests files. See https://github.com/pytorch/pytorch/pull/118318 for details.
```

Comment:
Is there is a way to suppress the lint? We'll probably want to allow some usages of torch.library.Library (just to test that API).

Reply:
It's disabled by default. When we enable it for some files, usual flake8 `noqa` comments can be used to ignore some lines.